lib/mongo/uri.rb

```diff
@@ -674,6 +690,26 @@ def ssl_verify(value)
       end
     end
 
+    # Parses the tlsAllowInvalidCertificates value.
+    #
+    # @param value [ String ] The tlsAllowInvalidCertificates value.
+    #
+    # @return [ true | false | nil ] The tlsAllowInvalidCertificates value parsed out, otherwise nil
+    #   (and a warning will be logged).
+    def tls_allow_invalid_certificates(value)
+      bool('tls_allow_invalid_invalid_certificates', value)
+    end
+
+    # Parses the tlsAllowInvalidHostnames value.
+    #
+    # @param value [ String ] The tlsAllowInvalidHostnames value.
+    #
+    # @return [ true | false | nil ] The tlsAllowInvalidHostnames value parsed out, otherwise nil
+    #   (and a warning will be logged).
```

Comment:
These should be marked `@api private` I think.

Reply:
None of the other option-parsing methods here have `@api private` so far. I can add it to all of them if you want.